pkg/apis/migration/migration.go

```diff
@@ -162,6 +162,7 @@ func GetVolumeMigrationService(ctx context.Context, volumeManager *cnsvolume.Man
 					metav1.NamespaceNone, config, true)
 				if err != nil {
 					log.Errorf("failed to create dynamic informer for volume migration CRD. Err: %v", err)
+					// TODO: exit?
```

Comment:
yes without informer driver can not function properly, we can exit if informer is critical for operation.

Reply:
Done